src/assets/scss/6-components/atoms/_buttons.scss

```diff
@@ -0,0 +1,175 @@
+.c-button {
+    @include font-style($font-primary, "small", #ffffff, 800, 16px);
+    @include padding(11px, 24px);
+    @include border-radius("base");
+    background-color: get-color-buttons("primary");
+    letter-spacing: 0.04em;
+    -webkit-appearance: none;
+    cursor: pointer;
+    border: none;
+
+    &:hover,
+    &.-hover {
+        background-color: get-color-buttons("primary-hover");
+    }
+    &:disabled {
+        cursor: not-allowed;
+        background-color: get-color-neutral("30");
+    }
+    &:active,
+    &.-active {
+        opacity: 0.8;
+        box-shadow: 0 0 0 3px rgba(0, 176, 202, 0.5);
+    }
+
+    > .c-icon {
+        > path {
+            fill: #ffffff;
+        }
+    }
+
+    &.-large {
+        @include padding(30px, 24px);
+        width: 100%;
+    }
+
+    &.-medium {
+        @include padding(11px, 24px);
```

Comment:
Is this intentionally redundant with the default?

Reply:
Good question. In porting some of these styles I have not had a chance to fully re-evaluate them. I'll at least leave a comment for later review.